docs/notebooks/opsim_notebook.ipynb

```diff
@@ -294,7 +315,7 @@
    "source": [
     "## Sampling\n",
     "\n",
-    "We can sample RA, dec, and time from an `OpSim` object using the `OpSimRADECSampler` node. This will select a random observation from the OpSim and then create a random (RA, dec) from near the center of that observation.\n",
+    "We can sample RA, dec, and time from an `OpSim` object using the `OpSimRADECSampler` node. This will select a random observation from the OpSim and then create a random (RA, Dec) from near the center of that observation.\n",
```

Comment:
```suggestion
    "We can sample RA, Dec, and time from an `OpSim` object using the `OpSimRADECSampler` node. This will select a random observation from the OpSim and then create a random (RA, Dec) from near the center of that observation.\n",
```
Not too sure on the standard for capitalizing Dec, but perhaps here too